kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -34,14 +34,14 @@ data:
           # In the absense of this configmap, default value for enabled is true
           enabled = true
        [log_collection_settings.enrich_container_logs]
-          # In the absense of this configmap, default value for enrich_container_logs is false
```

Comment:
This file had a bunch of invisible unicode characters (not sure what they were, probably some fancy formatting or something). I just removed them

Reply:
Could you check when you do a kubectl edit on the configmap it shows up in readable format after this change?

Reply:
Checked, the format isn't changed with this change (some parts of the file are non-human-readable, no idea why)